cygnus-twitter/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.telefonica.iot</groupId>
     <artifactId>cygnus-twitter</artifactId>
-    <version>3.5.0</version>
+    <version>3.5.0-SNAPSHOT</version>
```

Comment:
L60 should need also a fix?

(Comparing with https://github.com/telefonicaid/fiware-cygnus/pull/2338/files may help)